core/src/jsMain/kotlin/Bluetooth.kt

```diff
@@ -20,11 +20,11 @@ public fun CoroutineScope.requestPeripheral(
     options: Options,
     builderAction: PeripheralBuilderAction = {},
 ): Promise<Peripheral> = bluetooth
-    .requestDevice(options.toDynamic())
+    .requestDevice(options.toRequestDeviceOptions().also { console.info(it) })
```

Comment:
We should use the logger instead of spamming direct to console

Reply:
Oh yikes. Good catch! All the `console` logs were for debugging that I forgot to remove (or replace with `Log.*` entries). Will fix.